README.md

```diff
@@ -3379,7 +3379,167 @@ Response:
 }
 ```
 
+## GET /api/workflows/:version/callcaching/diff
 
+This endpoint returns the hash differences between 2 *completed* (successfully or not) calls.
+The following query parameters are supported:
+
+| Parameter |                                        Description                                        | Required |
+|:---------:|:-----------------------------------------------------------------------------------------:|:--------:|
+| workflowA | Workflow ID of the first call                                                             | yes      |
+| callA     | Fully qualified name of the first call. **Including workflow name**. (see example below)  | yes      |
+| indexA    | Shard index of the first call                                                             | no       |
```

Comment:
ToL Hmm, presumably this is `yes`, if the call was in a scatter, otherwise `no`? I don't know how to nicely purvey that though